### PR TITLE
feat(install): change 'Add ...' message

### PR DESCRIPTION
**Pull request by bartlomieju**
Changes messages printed by `deno add` from
`Add @oak/oak - jsr:@oak/oak^16.1.0` to
`Add jsr:@oak/oak16.1.0`

Screenshot 2024-08-08 at 13 59 35

### CODE REVIEW

**Comment by bartlomieju** [comment]
Other option:
Screenshot 2024-08-08 at 14 05 27

File: cli/tools/registry/pm.rs [hunk 1]
```diff
@@ -267,9 +267,8 @@ pub async fn add(
   let is_npm = config_file.is_npm();
   for selected_package in selected_packages {
     log::info!(
-      "Add {} - {}@{}",
-      crate::colors::green(&selected_package.import_name),
-      selected_package.package_name,
+      "Add {}@{}",
+      crate::colors::green(&selected_package.package_name),
```
Comment (ry) [inline cli/tools/registry/pm.rs:271]: can it be the resolved package version?

Comment (bartlomieju) [reply cli/tools/registry/pm.rs:271]: PTAL:
Screenshot 2024-08-08 at 15 28 51

Comment (ry) [reply cli/tools/registry/pm.rs:271]: Great!

**Review by ry — APPROVED** [review]

**Pull request merged**